string-manipulation-2/src/capitalize-word.ts

```diff
@@ -0,0 +1,13 @@
+/* exported capitalizeWord */
+function capitalizeWord(word: string): string {
+  if (word === 'jaVAsCrIPt' || word === 'javaScript' || word === 'JavascRipt') {
```

Comment:
If you convert the `word` to lowercase, you don't need to check for so many variations